.github/workflows/build.yml

```diff
@@ -24,3 +24,28 @@ jobs:
         run: yarn --immutable --immutable-cache
 
       - run: yarn ${{ matrix.command }}
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [18.x, 20.x]
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set Up node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node_version }}
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn --immutable --immutable-cache
+
+      - name: Run tests
+        run: yarn test
```

Comment:
did this change from test:integration to just test?

Reply:
yes - test runs both unit and integration tests